### PR TITLE
docs: update configuration

### PR DESCRIPTION
**Pull request by stevenjoezhang**
Currently, both the `_config.yml` and `README.md` of the landscape theme contain documentation for some configuration options, but neither is complete. I have moved all the configuration options and their explanations to `_config.yml`, and suggest that users directly copy this file to `_config.landscape.yml`. This way, it is more convenient to configure by referring to the hints inside, compared to consulting the README.

### CODE REVIEW

**Comment by stevenjoezhang** [comment]
`disqus_shortname` is missing in both `_config.yml` and `README.md`. Added in this pull request. See https://github.com/hexojs/hexo-theme-landscape/pull/40

**Comment by stevenjoezhang** [comment]
`index_widgets` was in `_config.yml` but I cannot find it anywhere else. See https://github.com/hexojs/hexo-theme-landscape/commit/67b97efdbf85376ea2bf229b2de9bd996ca98a2a and https://github.com/hexojs/hexo-theme-landscape/pull/21

File: README.md [hunk 1]
```diff
@@ -10,98 +10,56 @@ A brand new default theme for [Hexo].
...
+If you're using Hexo 5.0 or later, the simplest way to install is through npm:
 
-``` bash
-$ git clone --depth 1 https://github.com/hexojs/hexo-theme-landscape themes/landscape
+```
+npm i hexo-theme-landscape
 ```
 
-npm install:
+Local git install:
```
Comment (uiolee) [inline README.md:19]:
```suggestion
Install via git:
```

File: README.md [hunk 2]
```diff
@@ -10,98 +10,56 @@ A brand new default theme for [Hexo].
...
 ### Update
 
-``` bash
+Install the latest version through npm:
+
+```sh
+npm install hexo-theme-landscape@latest
+```
+
+Or update to the latest master branch:
```
Comment (uiolee) [inline README.md:45]:
```suggestion
Or update to the latest master branch via git:
```

**Review by uiolee — APPROVED** [review]
some suggestion FYI.

**Review by uiolee — APPROVED** [review]
lgtm

**Pull request merged**